batch_size: 4
detection_threshold: 0.5
resolution: 560
overlap: 100